pkg/controller/controller_instance_test.go

```diff
@@ -441,7 +499,7 @@ func TestReconcileInstance(t *testing.T) {
 
 	updatedInstance := assertUpdateStatus(t, actions[0], instance)
 	assertInstanceReadyTrue(t, updatedInstance)
-	assertInstanceDashboardURL(t, instance, testDashboardURL)
+	assertInstanceDashboardURL(t, updatedInstance, testDashboardURL)
```

Comment:
yeah, it was because we were mutating the cache.  On the plus side, one less cache mutation bug waiting in the wings.